### DISCUSSION TITLE
[Question]: csv and excel filetypes support

### DISCUSSION THREAD

**Discussion by sanyoganurag** [danny-avila/LibreChat#1965] [Q&A] [answered]
### What is your question?

Hi there,

Deployed with Az Open AI and App Service, and I was trying to upload csv and excel filetypes, is this supported?, however error is still "Unsupported file type", any other  config I need to follow ?

br
Sanyog

### More Details

Could not upload csv or excel file types

### What is the main subject of your question?

Installation, Endpoints

### Screenshots

<img width="584" alt="image" src="https://github.com/danny-avila/LibreChat/assets/15035914/8bd6875b-faa5-43ee-acaf-c361a2d3a4bb">

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment, marked as answer]
These filetypes are not yet supported natively for the Azure and regular OpenAI endpoints. CSV and Excel filetypes are only available for use with Assistants. This week I will release Azure support for Assistants, and in about 1-2 months there will be native RAG/file support for all endpoints.

**Reply by sanyoganurag** [reply]
Thanks for update Danny,

**Reply by itsolidude** [reply]
is the .tsx file type supported? There seems to be a file type conversion error with them.
